tests/numerics/fv/test_biot.py

```diff
@@ -27,13 +27,12 @@ def discretization_matrices(flow_keyword, mechanics_keyword):
     g.compute_geometry()
     stiffness = pp.FourthOrderTensor(np.ones(g.num_cells), np.ones(g.num_cells))
     bnd = pp.BoundaryConditionVectorial(g)
-    data = pp.initialize_default_data(
-        g,
+    data = pp.initialize_data(
         {},
         mechanics_keyword,
         specified_parameters=partial_update_parameters(stiffness, bnd, g.num_cells),
     )
-    data = pp.initialize_default_data(g, data, flow_keyword)
+    data = pp.initialize_data(data, flow_keyword)
```

Comment:
Is there any need to call initialize twice?

Reply:
Yes, this is just bad naming. What it does is initializes a sub-dictionary corresponding to the passed keyword. So we first initialize the "flow" subdictionary and then the "mechanics" subdictionary in data.

Reply:
I also don't think so, especially since we agree it's a user-facing api